packages/testing-helpers/index.d.ts

```diff
@@ -1,12 +1,13 @@
-export { html, unsafeStatic } from './lit-html';
+export { html, unsafeStatic } from './lit-html.js';
```

Comment:
Are extensions correct here?

Reply:
yes that is the local file... where we support dynamic tags (which is not yet available in official version) (note: this implementation is only suitable for testing as it's not performant)

Reply:
Should probably rename the tag and file then.

Reply:
it hopefully will land in lit-html right after 1.0 and then this would mean no change is needed at all for your test files... so "forward" compatible :p

I thought it was a worthwhile goal to not have to adopt every test file... with your replies I am not so sure anymore... anyways I think that change should be another PR... so if really needed pls open an issue :)